resources/source_vcf.vcf

```diff
@@ -0,0 +1,5 @@
+##fileformat=VCFv4.2
+#CHROM	POS ID  REF ALT QUAL	FILTER	INFO	FORMAT	HG001
+chr1	48	.	C	A	50	PASS	.	GT	1/1
+chr1	98	.	C	G	50	PASS	.	GT	1/1
+chr1	1078	.	A	G	50	PASS	.	GT	1/1
```

Comment:
Since this file is a _test_ resource, and not something actually required to run the pipeline in production, I'm thinking it might be better to move it to `test/resources/source_vf.vcf`

Reply:
It looks like this has been fixed partially. I can now see the `test/resources` folder with two files, `genome.fa` and `source_vcf.vcf`. However, in the repository root I can _also_ see the the `resources` folder with the same contents. Looks like it needs to be removed.

Reply:
Yes this will get fix in EVA-2001